plugins/inputs/docker/docker_test.go

```diff
@@ -298,8 +298,9 @@ func TestDockerGatherLabels(t *testing.T) {
 func TestDockerGatherInfo(t *testing.T) {
 	var acc testutil.Accumulator
 	d := Docker{
-		client:  nil,
-		testing: true,
+		client:         nil,
+		testing:        true,
+		TagEnvironment: []string{"ENVVAR1", "ENVVAR2", "ENVVAR3", "ENVVAR4", "ENVVAR5", "ENVVAR6", "ENVVAR7", "ENVVAR8", "ENVVAR9"},
```

Comment:
Please line wrap this after each element, not a hard rule but try to stick to about 80 chars per line.